---
title: About
layout: about
---
![Profile Image]({{ site.url }}/{{ site.picture }})

<p align="justify">Trained as biomedical engineer and software developer with skills in machine learning and big data analysis and visualization.</p>

<p align="justify">Working as a senior software developer and data analyst in the field of neuroscience. </p>

<p align="justify">Consulting on modern IT infrastructure implementation, organization and maintenance for big data storage and scientific computing.</p>

<h2>Skill set</h2>

<p  align="justify">The diversity of projects and techniques applied in neuroscience research demands usage of variety of tools. Each problem is approached with most suitable programming environment, e.g. agile and modern user interface is implemented with Qt/C++, embedded solutions like Arduino boards or National instrument cards are programmed with C language, data visualization is rendered with Matlab or Plotly via Python or R, machine learning techniques are used with Python and code and/or project maintenance is done with Git. As a software developer I am happy to use variety of tools, where one can compare and understand pros and cons of different APIs and is ready to apply most suited ones to the problem at stake.</p>

<div id="chart_skills"></div>

<h2>Experience</h2>

<p align="justify">Important part of software development in research field is to fully grasp and understand the problems. A thorough background in biology and neuroscience is required to smooth down the communication with scientists and abstract them out from the technical aspect of problem solving. Next big challenge is to keep track of modern techniques and equipment limits. Knowing instruments quality measures, synchronization demands and programmable capabilities gives you a head start in designing and implementing steady and reliable control or analysis software. Most of all, one needs to enjoy the thrill of solving challenging problems, application of novel algorithms and learning new libraries and programming paradigms. Self-driven urge of improvement is shaping day-to-day work.</p>

<div id="chart_experience"></div>

## Awards

[Friends of the MPI for Brain Research](https://brain.mpg.de/services/become-a-friend/) award for the best Ph.D. work in [2015](https://brain.mpg.de/fileadmin/user_upload/images/Friends/Newsletter_2_2015.pdf).

<h2>Publications</h2>

### 2018
Tosches, M.A., Yamawaki, T.M., Naumann, R.K., Jacobi, A.A., **Tushev, G.** and Laurent, G. (2018). Evolution of pallium, hippocampus and cortical cell types revealed by single-cell transcriptomics in reptiles. ***Science***,
PMID: [29724907](https://www.ncbi.nlm.nih.gov/pubmed/29724907)

**Tushev, G.**, Glock, C., Heumueller, M., Biever, A., Jovanovic, M., and Schuman, E.M. (2018). Alternative 3â€™UTRs modify the localization, regulatory potential, stability, and plasticity of mRNAs in neuronal compartments. ***Neuron***, PMID: [29656876](https://www.ncbi.nlm.nih.gov/pubmed/29656876/)

### 2017
Vlatkovic, I., Sambandan, S., **Tushev, G.**, Wang, M., Epstein, I., Glock, C., Fuerst, N., Cajigas, I.J., and Schuman, E.M. (2017). Poly(A) Binding Protein Nuclear 1 regulates the polyadenylation of key synaptic plasticity genes and plays a role in homeostatic plasticity. ***bioRxiv***, [121194](https://www.biorxiv.org/content/early/2017/03/27/121194)

Akbalik, G., Langebeck-Jensen, K., **Tushev, G.**, Sambandan, S., Rinne, J., Epstein, I., Cajigas, I.J., Vlatkovic, I., Schuman, E.M. (2017). Visualization of newly synthesized neuronal RNA in vitro and in vivo using click-chemistry. ***RNA Biology***, PMID: [27801616](https://www.ncbi.nlm.nih.gov/pubmed/27801616/)

Sambandan, S., Akbalik, G., Kochen, L., Rinne, J., Kahlstatt, J., Glock, C., **Tushev, G.**, Alvarez-Castelao, B., Heckel, A., and Schuman, E.M. (2017). Activity-dependent spatially localized miRNA maturation in neuronal dendrites. ***Science***, PMID: [28183980](https://www.ncbi.nlm.nih.gov/pubmed/28183980)

### 2016
Hanus, C., Geptin, H., **Tushev, G.**, Garg, S., Alvarez-Castelao, B., Sambandan, S., Kochen, L., Hafner, A.S., Langer, J.D., Schuman, E.M. (2016). Unconventional secretory processing diversifies neuronal ion channel properties. ***eLife***, PMID: [27677849](https://www.ncbi.nlm.nih.gov/pubmed/27677849)

**Tushev, G.** & Schuman, E.M. (2016). Rethinking functional segregation: Gradients of gene expression in area CA1. ***Neuron***, PMID: [26796687](https://www.ncbi.nlm.nih.gov/pubmed/26796687)

### 2015
You, X., Vlatkovic, I., Babic, A., Will, T.J., Epstein, I., **Tushev, G.**, Akbalik, G., Wang, M., Glock, C., Quedenau, C., Wang, X., Hou, J., Liu, H., Sun, W., Sambandan, S., Chen, T., Schuman, E.M. and Chen, W.  (2015) Neural circular RNAs are derived from synaptic genes and regulated by development and plasticity. ***Nature Neuroscience***, PMID: [25714049](https://www.ncbi.nlm.nih.gov/pubmed/25714049)

### 2014
Will, T.J., **Tushev, G.**, Kochen, L., Nassim-Assir, B., Cajigas, I.J., tom Dieck, S., and Schuman, E.M. (2013). Deep-Sequencing and High-Resolution Imaging Reveal Compartment-Specific Localization of Bdnf mRNA in Hippocampal Neurons. ***Science Signaling***, PMID: [24345682](https://www.ncbi.nlm.nih.gov/pubmed/24345682)

### 2013
Epstein, I., **Tushev, G.**, Will, T.J., Vlatkovic, I., Cajigas, I.J., Schuman, E.M. (2013). Alternative polyadenylation and differential expression of Shank mRNAs in the synaptic neuropil. ***Phil. Trans. R. Soc. B***, PMID: [24298140](https://www.ncbi.nlm.nih.gov/pubmed/24298140)

Hinz, F.I., **Tushev, G.**, Aizenberg, M., and Schuman, E.M. (2013). Protein synthesis-dependent associative long-term memory in larval zebrafish. ***Journal of Neuroscience***, PMID: [24068805](https://www.ncbi.nlm.nih.gov/pubmed/24068805)

### 2012
Cajigas, I.J., **Tushev, G.**, Will, T.J., tom Dieck, S., Fuerst, N., and Schuman, E.M.: The Local Transcriptome in the Synaptic Neuropil Revealed by Deep Sequencing and High-Resolution Imaging. ***Neuron***, PMID: [22578497](https://www.ncbi.nlm.nih.gov/pubmed/22578497)
